### DISCUSSION TITLE
Custom page background?

### DISCUSSION THREAD

**Discussion by Poolitzer** [xournalpp/xournalpp#3488] [Q&A]
Hello,

How can I do a custom page? I basically want to add a vertical line to a graph background.

Thanks for your help-

**Comment by personalizedrefrigerator** [comment]
Try setting the page's background to an image of what you want:
![Journal > Paper Background > Image menu sets to a custom background](https://user-images.githubusercontent.com/46334387/138614101-73b1d292-9b5b-48c5-bbe2-7b18ec9fefea.png)

**Reply by Poolitzer** [reply]
okay, but then I would need to make an image. Pretty sure that will blow up the file unnecessarily, while all I want is Graph with vertical line.

Can I lock a line so I dont erase it by accident?

Edit: Yes I can, I can use a second layer. Ha, that was easy.
